packages/headless/src/controllers/facets/automatic-facet/headless-automatic-facet.ts

```diff
@@ -7,70 +7,29 @@ import {logFacetClearAll} from '../../../features/facets/facet-set/facet-set-ana
 import {getAnalyticsActionForToggleFacetSelect} from '../../../features/facets/facet-set/facet-set-utils';
 import {FacetValue} from '../../../features/facets/facet-set/interfaces/response';
 import {executeSearch} from '../../../features/search/search-actions';
-import {
-  Controller,
-  buildController,
-} from '../../controller/headless-controller';
+import {buildController} from '../../controller/headless-controller';
+import {AutomaticFacet} from '../automatic-facet-generator/headless-automatic-facet-generator';
 
 /**
- * A scoped and simplified part of the headless state that is relevant to the `Automatic Facet` controller.
- */
-export interface AutomaticFacetState {
-  /**
-   * The automatic facet field.
-   */
-  field: string;
-  /**
-   * The automatic facet label.
-   */
-  label: string;
-  /**
-   * The automatic facet values.
-   */
-  values: FacetValue[];
-}
-/**
- * The props for the `AutomaticFacet` controller.
+ * @internal
```

Comment:
If the props / builder aren't exported, do we even need to document them?

Reply:
I think its fine to leave the comments.